src/core/types/non_empty_max_n_bytes.rs

```diff
@@ -145,6 +146,11 @@ decl_non_empty_max_n_bytes!(
     64
 );
 
+decl_non_empty_max_n_bytes!(
+    /// 32 bytes, typically used as the Bultin for `BIP39Entropy`.
```

Comment:
TODO: Change misleading comment. `BIP39Entropy` was UniFFI convertible for a while, not it is not.